test/kubemark/start-kubemark.sh

```diff
@@ -235,25 +337,52 @@ if [ "${CLOUD_PROVIDER}" = "aws" ]; then
 else
   echo "Master IP: ${MASTER_IP}"
 fi
-echo "Kubeconfig for kubemark master is written in ${LOCAL_KUBECONFIG}"
+echo "Kubeconfig for kubemark master is written in ${LOCAL_KUBECONFIG_TMP}"
+
+# all kubectl OPs go via the proxy
+#
+### internally the TP configures are set with proxy URL, simply copy for POC phase for now
+###
+if [[ "${SCALEOUT_CLUSTER:-false}" == "true" ]]; then
+  cp -f ${TP_KUBECONFIG} ${PROXY_KUBECONFIG_SAVED}
+fi
 
 sleep 5
 echo -e "\nListing kubeamrk cluster details:" >&2
 echo -e "Getting total nodes number:" >&2
-"${KUBECTL}" --kubeconfig="${LOCAL_KUBECONFIG}" get node | wc -l
+"${KUBECTL}" --kubeconfig="${TP_KUBECONFIG}" get node | wc -l
 echo
 echo -e "Getting total hollow-nodes number:" >&2
-"${KUBECTL}" --kubeconfig="${LOCAL_KUBECONFIG}" get node | grep "hollow-node" | wc -l
+"${KUBECTL}" --kubeconfig="${TP_KUBECONFIG}" get node | grep "hollow-node" | wc -l
 echo
 echo -e "Getting endpoints status:" >&2
-"${KUBECTL}" --kubeconfig="${LOCAL_KUBECONFIG}" get endpoints -A
+"${KUBECTL}" --kubeconfig="${TP_KUBECONFIG}" get endpoints -A
 echo
 echo -e "Getting workload controller co status:" >&2
-"${KUBECTL}" --kubeconfig="${LOCAL_KUBECONFIG}" get co
+"${KUBECTL}" --kubeconfig="${TP_KUBECONFIG}" get co
 echo
 echo -e "Getting apiserver data partition status:" >&2
-"${KUBECTL}" --kubeconfig="${LOCAL_KUBECONFIG}" get datapartition
+"${KUBECTL}" --kubeconfig="${TP_KUBECONFIG}" get datapartition
 echo
 echo -e "Getting ETCD data partition status:" >&2
-"${KUBECTL}" --kubeconfig="${LOCAL_KUBECONFIG}" get etcd
+"${KUBECTL}" --kubeconfig="${TP_KUBECONFIG}" get etcd
 echo
+
+### for multiple tenant tests, set up the tenants on desired TP clusters
```

Comment:
As mentioned before, line 371-388 can be removed.

Reply:
this is also what we currently have in scale-out-poc TOT